### DISCUSSION TITLE
Can i set style for a single page

### DISCUSSION THREAD

**Discussion by Lix1993** [ObsidianPublisher/obsidian-github-publisher#167] [Ideas]
eg: no search,navbar,sidebar for index page

**Comment by Mara-Li** [comment]
Hello.
For nav & sidebar, you can check here: [usage](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#usage)
For search… I dunno. Maybe with using [css class](https://obsidian-publisher.netlify.app/changelog/css%20class/).
